src/main/java/com/gregtechceu/gtceu/common/item/tool/ToolEventHandlers.java

```diff
@@ -189,4 +191,32 @@ public static void onAnvilUpdateEvent(AnvilUpdateEvent event) {
             event.setCanceled(true);
         }
     }
+
+    public static Collection<ItemEntity> onPlayerKilledEntity(ItemStack tool, Player player, Collection<ItemEntity> drops) {
+        CompoundTag behaviorTag = ToolHelper.getBehaviorsTag(tool);
+
+        if (behaviorTag.getBoolean(ToolHelper.RELOCATE_MOB_DROPS_KEY)) {
+            Iterator<ItemEntity> dropItr = drops.iterator();
+
+            while (dropItr.hasNext()) {
+                ItemStack dropStack = dropItr.next().getItem();
+                ItemEntity drop = new ItemEntity(EntityType.ITEM, player.level());
+
+                drop.setItem(dropStack);
+                if (fireItemPickupEvent(drop, player) == -1 || player.addItem(dropStack)) {
+                    dropItr.remove();
+                }
+            }
+        }
+        return drops;
+    }
+
+    @SubscribeEvent
+    public static void onPlayerKilledEntity(LivingDeathEvent event) {
+        Entity player = event.getSource().getEntity();
+        if (player instanceof Player) {
+            ToolEventHandlers.onPlayerKilledEntity(((Player) player).getMainHandItem(), (Player) player, event.getEntity().captureDrops());
```

Comment:
this is wrong. you want `event.getDrops()` instead of `event.getEntity().captureDrops()`.